specs/PrintToPdf.md

```diff
@@ -0,0 +1,294 @@
+/// currently supported.
+[uuid(30964a64-2e92-4604-9eaa-3afcd3a015dc), object, pointer_default(unique)]
+interface ICoreWebView2PrintSettings : IUnknown {
+
+  /// The orientation can be portrait or landscape. The default orientation is
+  /// portrait. See `COREWEBVIEW2_PRINT_ORIENTATION`.
+  [propget] HRESULT Orientation(
+    [out, retval] COREWEBVIEW2_PRINT_ORIENTATION* orientation);
+
+  /// Sets the `Orientation` property.
+  [propput] HRESULT Orientation(
+      [in] COREWEBVIEW2_PRINT_ORIENTATION orientation);
+
+  /// The scale factor is a value between 0.1-2.0. The default is 1.0.
+  [propget] HRESULT ScaleFactor([out, retval] double* scaleFactor);
+
+  /// Sets the `ScaleFactor` property.
+  [propput] HRESULT ScaleFactor([in] double scaleFactor);
+
+  /// The page width in inches. The default width is 8.5 inches.
+  [propget] HRESULT PageWidth([out, retval] double* pageWidth);
+
+  /// Sets the `PageWidth` property.
+  [propput] HRESULT PageWidth([in] double pageWidth);
+
+  /// The page height in inches. The default height is 11 inches.
+  [propget] HRESULT PageHeight([out, retval] double* pageHeight);
+
+  /// Sets the `PageHeight` property.
+  [propput] HRESULT PageHeight([in] double pageHeight);
+
+  /// The top margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginTop([out, retval] double* marginTop);
+
+  /// Sets the `MarginTop` property.
+  [propput] HRESULT MarginTop([in] double marginTop);
+
+  /// The bottom margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginBottom([out, retval] double* marginBottom);
+
+  /// Sets the `MarginBottom` property.
+  [propput] HRESULT MarginBottom([in] double marginBottom);
+
+  /// The left margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginLeft([out, retval] double* marginLeft);
+
+  /// Sets the `MarginLeft` property.
+  [propput] HRESULT MarginLeft([in] double marginLeft);
+
+  /// The right margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginRight([out, retval] double* marginRight);
+
+  /// Set the `MarginRight` property.
+  [propput] HRESULT MarginRight([in] double marginRight);
+
+  /// `TRUE` if backgrounds should be printed. The default value is `FALSE`.
+  [propget] HRESULT ShouldPrintBackgrounds(
+      [out, retval] BOOL* shouldPrintBackgrounds);
+
+  /// Set the `ShouldPrintBackgrounds` property.
+  [propput] HRESULT ShouldPrintBackgrounds([in] BOOL shouldPrintBackgrounds);
+
+  /// `TRUE` if only the selection should be printed. The default value is `FALSE`.
+  [propget] HRESULT ShouldPrintSelectionOnly(
+      [out, retval] BOOL* shouldPrintSelectionOnly);
+
+  /// Set the `ShouldPrintSelectionOnly` property.
+  [propput] HRESULT ShouldPrintSelectionOnly(
+      [in] BOOL shouldPrintSelectionOnly);
+
+  /// `TRUE` if header and footer should be printed. The default value is `FALSE`.
```

Comment:
It would be good to say what header and footer are in this context. I had thought at first this was something from the web page until I read the next couple of properties. Now I assume the header only contains the header title and the footer only contains the footer URI, is that correct?